src/main/java/com/bootcamp/demo/controller/TransactionController.java

```diff
@@ -48,4 +54,20 @@ public TransactionController(TransactionService transactionService) {
     public List<Transaction> getTransactionDetails() {
         return transactionService.getTransactionDetails();
     }
+
+    @GetMapping
+    public ModelAndView getTransactionHistory() {
```

Comment:
Note: A transaction history feature was already implemented in Pull Request https://github.com/remus-lupin-team/payments-jb/pull/23